homeassistant/components/tuya/strings.json

```diff
@@ -18,61 +18,270 @@
+      },
+      "pressure": {
+        "name": "[%key:component::sensor::entity_component::pressure::name%]"
+      },
+      "feeding": {
+        "name": "Feeding"
+      },
+      "drop": {
+        "name": "Drop"
+      },
+      "tilt": {
+        "name": "Tilt"
+      }
+    },
+    "button": {
+      "reset_duster_cloth": {
+        "name": "Reset duster cloth"
+      },
+      "reset_edge_brush": {
+        "name": "Reset edge brush"
+      },
+      "reset_filter": {
+        "name": "Reset filter"
+      },
+      "reset_map": {
+        "name": "Reset map"
+      },
+      "reset_roll_brush": {
+        "name": "Reset roll brush"
+      },
+      "snooze": {
+        "name": "Snooze"
+      }
+    },
+    "cover": {
+      "curtain_2": {
+        "name": "Curtain 2"
+      },
+      "curtain_3": {
+        "name": "Curtain 3"
+      },
+      "door": {
+        "name": "[%key:component::cover::entity_component::door::name%]"
+      },
+      "door_2": {
+        "name": "Door 2"
+      },
+      "door_3": {
+        "name": "Door 3"
+      }
+    },
+    "light": {
+      "backlight": {
+        "name": "Backlight"
+      },
+      "light": {
+        "name": "[%key:component::light::title%]"
+      },
+      "light_2": {
+        "name": "Light 2"
+      },
+      "light_3": {
+        "name": "Light 3"
+      },
+      "night_light": {
+        "name": "Night light"
+      }
+    },
+    "number": {
+      "time": {
+        "name": "Time"
+      },
+      "temperature_after_boiling": {
+        "name": "Temperature after boiling"
+      },
+      "heat_preservation_time": {
+        "name": "Heat preservation time"
+      },
+      "feed": {
+        "name": "Feed"
+      },
+      "voice_times": {
+        "name": "Voice times"
+      },
+      "sensitivity": {
+        "name": "Sensitivity"
+      },
+      "near_detection": {
+        "name": "Near detection"
+      },
+      "far_detection": {
+        "name": "Far detection"
+      },
+      "water_level": {
+        "name": "Water level"
+      },
+      "powder": {
+        "name": "Powder"
+      },
+      "cook_temperature": {
+        "name": "Cook temperature"
+      },
+      "cook_time": {
+        "name": "Cook time"
+      },
+      "cloud_recipe": {
+        "name": "Cloud recipe"
+      },
+      "volume": {
+        "name": "Volume"
+      },
+      "minimum_brightness": {
+        "name": "Minimum brightness"
+      },
+      "maximum_brightness": {
+        "name": "Maximum brightness"
+      },
+      "minimum_brightness_2": {
+        "name": "Minimum brightness 2"
+      },
+      "maximum_brightness_2": {
+        "name": "Maximum brightness 2"
+      },
+      "minimum_brightness_3": {
+        "name": "Minimum brightness 3"
+      },
+      "maximum_brightness_3": {
+        "name": "Maximum brightness 3"
+      },
+      "move_down": {
+        "name": "Move down %"
+      },
+      "move_up": {
+        "name": "Move up %"
+      },
```

Comment:
We should remove the percentage and add the unit of measurement property instead.

This originates from the time number entities didn't have a unit of measurement.